css_files/rules.css

```diff
@@ -0,0 +1,13 @@
+
+.rule{
+    font-size: 1.6rem;
+    margin-bottom: var(--sm-space);
+}
+
+.btn-rules{
+    font-size: 1.6rem;
```

Comment:
create variables for font-sizes as well, to make it consistent

Reply:
yes